app/src/main/java/net/gsantner/markor/util/AppSettings.java

```diff
@@ -170,6 +170,10 @@ public boolean isMarkdownDisableCodeBlockHighlight() {
         return getBool(R.string.pref_key__markdown__disable_code_block_highlight, false);
     }
 
+    public boolean isMarkdownAutoUpdateList() {
+        return getBool(R.string.pref_key__markdown__auto_renumber_ordered_list, false);
```

Comment:
I'm fine with it being easy to enable/disable in code, but we don't need a settings option for it. It's fine to have this appsettings method and just return true. And have no settings.xml option.